### DISCUSSION TITLE
📣 NEW Blog! Learn how to deal with label errors in the popular IMDb movie review dataset.

### DISCUSSION THREAD

**Discussion by weijinglok** [cleanlab/cleanlab#254] [Show and tell]
Learn how to deal with label errors in the popular IMDb movie review dataset. Our blog teaches you how to:

1. Construct a TensorFlow/Keras neural net for text classification and train it via cross-validation.
2. Automatically find badly labeled data with cleanlab. 
3. Use cleanlab to train a more robust version of the same text classifier that is more resilient to noisily labeled data.

Authored by [Wei Jing Lok](https://www.linkedin.com/in/weijinglok/) and @jwmueller
🔗 Blog Post + Google Colab: [cleanlab.ai/blog/label-errors-text-datasets/](https://cleanlab.ai/blog/label-errors-text-datasets/)

Other Links:
🔗 Slack: [join.slack.com/t/cleanlab-community/shared_invite/zt-18q7qlwic-o9wyltbbGBDCiI4U~BaPug](https://join.slack.com/t/cleanlab-community/shared_invite/zt-18q7qlwic-o9wyltbbGBDCiI4U~BaPug)
🔗 Twitter: [twitter.com/CleanlabAI](http://twitter.com/CleanlabAI)
🔗 Facebook: [facebook.com/CleanlabAI](http://facebook.com/CleanlabAI)
🔗 LinkedIn: [linkedin.com/company/cleanlab](https://www.linkedin.com/company/cleanlab)
